### ISSUE TITLE
32bit device and "No Apple device in DFU Mode detected" error

### ISSUE THREAD

**Issue by nemmeno667** [MatthewPierson/Vieux#186]
**Describe the issue**

Every time I try to downgrade the iPad mini (iPad2,7, 32bit) to iOS 8.4.1, I get the message

"ERROR: No Apple device in DFU Mode 0x1227 detected after 30.00 second timeout. Exiting. Place the device into DFU mode again and re-run Vieux"

even if it's a 32bit device and it should be in normal mode as asked. Of course, trying to run vieux when iPad is already in DFU ends with error because of the impossibility to connect to the device in SSH.

iPad is jailbroken with Phoenix every time before run the script; OpenSSH package installed through Cydia on the iPad

**Screenshots**

<img width="1263" alt="Screenshot 2020-07-29 at 00 31 52" src="https://user-images.githubusercontent.com/68928677/88729647-abdb4500-d134-11ea-9a83-68f5da05c937.png">

<img width="1263" alt="Screenshot 2020-07-29 at 00 31 35" src="https://user-images.githubusercontent.com/68928677/88729700-c7465000-d134-11ea-8958-23d04109dc3e.png">

**Computer (please complete the following information):**

 - MacBook Pro Early 2013
 - OS X 10.14.6
 - Python Version 3.8.5
 - Vieux Version 1.0.1

**iOS Device (please complete the following information):**

 - Device: iPad2,7
 - Desired Version: iOS 8.4.1
 - Current Version: iOS 9.3.6

**Comment by MatthewPierson** [comment]
Are you pressing the home button and re-plugging in the device in when prompted?

**Comment by nemmeno667** [comment]
yes, of course. right after I press the home button and unplug-replug the device iTunes opens and the script goes to the next step.

sometimes it also happens the error change; it says “Getting SHSH failed”...
